### DISCUSSION TITLE
Please help me enable macOS window border separators

### DISCUSSION THREAD

**Discussion by josepheiba** [raphamorim/rio#445] [Q&A] [answered]
<img width="455" alt="Screenshot 2024-02-02 at 4 05 14 PM" src="https://github.com/raphamorim/rio/assets/101673502/cea0a3c9-28f7-49be-8ba8-07c71cd0966c">

hello, any idea how to enable the window separators/borders on macOS, all other Mac apps have these borders that help separate windows when they all using the same colors

**Comment by raphamorim** [comment, marked as answer]
These borders will be available from 0.1.0 version, after migrate from winit to wa on macos.

Below you can see rio 0.1.x and 0.0.x

![image](https://github.com/raphamorim/rio/assets/3630346/57a02b77-3022-4075-81e4-210cd0b2fa46)

**Reply by josepheiba** [reply]
thank you, I'm looking forward
